apps/frontend/src/app/3_organisms/TransactionHistoryFrame/TransactionHistoryFrame.tsx

```diff
@@ -35,8 +37,15 @@ import { useGetTroves } from './hooks/useGetTroves';
 const DEFAULT_PAGE_SIZE = 10;
 const liquidationReserveAmount = 20;
 
-export const TransactionHistoryFrame: FC = () => {
+interface TransactionHistoryFrameProps {
+  account: string;
```

Comment:
I am just thinking - do you need to pass it as a prop? Will we have a use case where we'll want to show the transaction history of someone else's account and not yours? If not, you can delete this type and just use `useAccount` hook in the component.